### PR TITLE
Send notifications

### PR DESCRIPTION
**Pull request by markusressel**
added notifier class
removed ADMIN, send notification to all users in admin list instead
added __main__

### CODE REVIEW

File: frundenbot/main.py [hunk 1]
```diff
@@ -125,15 +130,31 @@ def _callback_help(self, update: Update, context: CallbackContext):
...
         context.bot.sendMessage(chat_id=update.message.chat_id,
-                                text='{}\nÜbrigens kannst du mit /mate nachgucken, ob es noch Getränke gibt.'.format(cache))
+                                text='{}\nÜbrigens kannst du mit /mate nachgucken, ob es noch Getränke gibt.'.format(
+                                    cache))
+
+    NOTIFY_TIME = Summary('notify_seconds',
+                          'Time spent executing /notify handler')
+
+    @NOTIFY_TIME.time()
+    @command(name='notify', description='Get a notification when the Freitagsrunde opens up')
```
Comment (ekeih) [inline frundenbot/main.py]:
```suggestion
    @command(name='notify', description='Get a notification when the Freitagsrunde opens up.')
```

File: frundenbot/notifier.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+
+class Notifier:
+    """
+    Used to send notifications when the "open" status changes
+    """
+
+    # set of chat_ids that have registered for a notification
+    chat_ids = set()
+    # the last
```
Comment (ekeih) [inline frundenbot/notifier.py]: This looks a bit like you wanted to write more? :thinking:

Comment (markusressel) [reply frundenbot/notifier.py]: obsolete

**Comment by ekeih** [comment]
The s3 support doesn't work if the object doesn't exist yet. I am afraid the existing fail-safes you put in the code for that situation are not enough yet.
I will send you some s3 credentials which you can use to test this.

**Pull request merged**